FlowCrypt/Extensions/URLSessionExtension.swift

```diff
@@ -24,7 +24,7 @@ extension URLSession {
             self.dataTask(with: urlRequest) { data, response, error in
                 let res = response as? HTTPURLResponse
                 let status = res?.statusCode ?? GeneralConstants.Global.generalError
-                print("URLSession.call status:\(status) ms:\(start.millisecondsSince) \(urlRequest.httpMethod ?? "GET") \(urlRequest.url?.absoluteString ?? "??")")
+//                debugPrint("URLSession.call status:\(status) ms:\(start.millisecondsSince) \(urlRequest.httpMethod ?? "GET") \(urlRequest.url?.absoluteString ?? "??")")
```

Comment:
I would still print these, it's useful to see calls / results, unless you have some other ways to inspect http requests?

Reply:
It will spam a lot. 
Maybe there is an idea to have different levels for logging? 
Like UI logs, Imap logs, session logs and so on.
And also be able to set a list of logs to inspect.